src/vs/workbench/browser/parts/compositeBarActions.ts

```diff
@@ -151,18 +153,17 @@ export class ActivityActionItem extends BaseActionItem {
 
 	protected updateStyles(): void {
 		const theme = this.themeService.getTheme();
-
+		const colors = this.options.colors(theme);
 		// Label
 		if (this.label && this.options.icon) {
-			const background = theme.getColor(this.options.colors.backgroundColor);
-
+			const background = this._action.checked ? colors.activeBackgroundColor : colors.backgroundColor;
```

Comment:
@sandy081 should we also set the foreground if no icon is provided and the thing is checked? Looks like we do that via a CSS rule ([here](https://github.com/Microsoft/vscode/blob/e37c7306490f6f3f1ebf5580767f84ce74fc70a8/src/vs/workbench/browser/parts/panel/panelPart.ts#L325)) but it seems it should rather be defined here.

Reply:
Yes, this should be done here instead in CSS. That makes code cleaner.